database/data-generator.js

```diff
@@ -107,6 +111,22 @@ const generateElasticBookingDetails = async (start = 1000000, finish = 2000000)
   }
 };
 
-generateBookingDetails();
-generateBulkBookingDetails();
-generateElasticBookingDetails();
+const generateSQSBookingDetails = async (start = 2002000, finish = 2003000) => {
+  let bulk = [];
+  for (let listingId = start; listingId < finish; listingId += 1) {
+    for (let startingDay = 5; startingDay <= 129; startingDay += 7) {
```

Comment:
These numbers seem arbitrary, especially since a start of 5, increment of 7 and end of 129 do not add up.

Reply:
The 5 was used because I wanted to adjust it so that higher bookings would land on the weekends, which meant shifting the dates by 4 which is a bit of a cludgy solution. This is definitely something to fix/refactor and I'll make a trello ticket for it.